src/widgets/modals/StocktakeBatchModal.js

```diff
@@ -0,0 +1,336 @@
+/* eslint-disable react/require-default-props */
+/* eslint-disable react/forbid-prop-types */
```

Comment:
Why?

Reply:
Thanks, I needed to make an issue for this. Objects and arrays are forbidden prop types according to our ESLint rules. Default props was laziness. 8b8530020599f302b2b809513ecb2aa6a25faa77

Reply:
Hmm I suppose ESLint probably screaming to define the shape of the prop rather than use object/array? It's kind of out own laziness, but then again it's fairly tedious and not totally convinced we'll get much more value out of the effort. @wlthomson was driving eslint changes so he might have a say, though probs in #845 you made so happy to resolve this.